tests/ios/xcodeproj/Single-Static-Framework-Project.xcodeproj/bazelinstallers/lldb-settings.sh

```diff
@@ -13,11 +13,7 @@ set -euo pipefail
 #
 # The platform settings one points the current working directory (cwd) to workspace root.
 # This is needed for features relying on lldb remote debugging, such as `oso_prefix_is_pwd`.
-#
-# NOTE: In order to use this, add this line to `~/.lldbinit`:
-#
-#     command source ~/.lldbinit-source-map
-cat <<-END > ~/.lldbinit-source-map
+cat > $BAZEL_LLDB_INIT_FILE <<-END
```

Comment:
Reason for this change:

```
3.6.6 Here Documents
...
No parameter and variable expansion, command substitution, arithmetic expansion, or filename expansion is performed on word.
```

See https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Here-Documents